### ISSUE TITLE
foursquare_node.rb:13 - .blank? method requires the Rails Object

### ISSUE THREAD

**Issue by sordyl** [ronaldwan/foursquare-api#1]
I believe the .blank? method is introduced by Rails (http://api.rubyonrails.org/classes/Object.html#method-i-blank%3F)  but Rails is not a dependency for this gem.

I'm testing via irb on ruby 1.8.7 and the example from the README fails:

> > require "foursquare"

=> true

> > venue = Foursquare::Venue.new("access token")

=> #<Foursquare::Venue:0x1013ac468 @base_url="https://api.foursquare.com:443/v2/", @access_token="access token">

> > venue.search({:ll => "37.792694,-122.409325"})

NoMethodError: undefined method `blank?' for "access token":String
    from /Library/Ruby/Gems/1.8/gems/foursquare-api-0.0.5/lib/foursquare_node.rb:13:in`perform_graph_request'
    from /Library/Ruby/Gems/1.8/gems/foursquare-api-0.0.5/lib/foursquare_venue.rb:31:in `search'
    from (irb):3

**Comment by ronaldwan** [comment]
Good catch!  thanks for letting me know.  Ill patch this up in the next few days when im not on the road anymore.

Thanks again!
